controlplane/kubeadm/internal/machinefilters/machine_filters.go

```diff
@@ -154,16 +152,15 @@ func IsReady() Func {
 
 // ShouldRolloutAfter returns a filter to find all machines
 // that should rollout after the given time.
-func ShouldRolloutAfter(t *metav1.Time) Func {
-	now := metav1.Now()
+func ShouldRolloutAfter(now, upgradeAfter *metav1.Time) Func {
```

Comment:
nit, `now` seems a bit misleading if we are allowing it to be passed in.
```suggestion
func ShouldRolloutAfter(t, upgradeAfter *metav1.Time) Func {
```

Reply:
Yeah, I had this as t0 and t1 but then the unit tests for this method were calling things `upgradeAfter` etc, and even the method itself being called `ShouldRolloutAfter` instead of `IsBefore` or something leaks in all sorts of caller knowledge and I thought it was maybe more straightforward. I agree though, I'll change this and maybe change the name too.